charts/prometheus/values.yaml

```diff
@@ -1311,7 +1311,33 @@ alertmanager:
   enabled: true
 
   persistence:
-    size: 2Gi
+    ## If true, storage will create or use Persistence Volume
+    ## If false, storage will use emptyDir
+    ##
+    enabled: true
+
+    ## Custom annotations for the PVC created by the alertmanager StatefulSet.
+    ## Useful for configuring storage provider options such as disk type, KMS encryption keys, or custom volume name prefixes.
+    annotations: {}
+
+    ## Custom labels for the PVC created by the alertmanager StatefulSet.
+    ## Useful for selecting, grouping, and organizing so that they can be queried or targeted in deployments, policies, etc.
+    labels: {}
+
+    ## Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ## set, choosing the default provisioner.
+    ##
+    # storageClass: "-"
+    accessModes:
+      - ReadWriteOnce
+    size: 50Mi
```

Comment:
Changing a default value is a major change. Users would need to know that it has changed via an upgrade note.
An upgrade without specifying the field's custom (user's) value will fail as the corresponding field in the statefulset is immutable. Again, users would have to be told what needs to be done before upgrading.

Please, keep the previous value. With all respect, there is no reason for changing it.

Reply:
In my opinion, that would be unnecessary. The prometheus chart chose different defaults as charts often do for their subcharts.